index.html

```diff
@@ -9,19 +9,25 @@
   </head>
 
   <body>
+    <label>
+      <input type="checkbox" id="filterConsent" />
+      Show only courses that require consent
+    </label>
+
     <table>
       <thead>
         <tr>
-          <td>Course</td>
+          <td style="background-color: powderblue">Course</td>
+          <td style="background-color: powderblue">Consent</td>
```

Comment:
We should separate our styling from our HTML markup. These styles shouldn't be inlined here.